pkgs/by-name/no/nominatim-ui/package.nix

```diff
@@ -15,6 +37,43 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-TliTWDKdIp7Z0uYw5P65i06NQAUNwNymUsSYrihVZFE=";
   };
 
+  offlineCache = fetchYarnDeps {
+    yarnLock = "${finalAttrs.src}/yarn.lock";
+    hash = "sha256-IqwsXEd9RSJhkA4BONTJT4xYMTyG9+zddIpD47v6AFc=";
+  };
+
+  nativeBuildInputs = [
+    fixup-yarn-lock
+    nodejs
+    yarn
+  ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    export HOME=$(mktemp -d)
```

Comment:
Looks like `writableTmpDirAsHomeHook` in `nativeBuildInputs` should do the trick. (I'm not sure why so many packages use what is presumably a non-reproducible pattern during builds.)

Reply:
Done.